docs/source/exit_codes.rst

```diff
@@ -0,0 +1,20 @@
+.. _exit-codes:
+
+Exit codes
+===========
+
+Mypy, like all programs, returns an
+`exit code <https://en.wikipedia.org/wiki/Exit_status>`_ to the process or shell
+that invokes it. The meaning of these codes is thus:
+
+* 0: no type errors
+
+* 1: there are some type errors
+
+* 2: a crash, bad arguments or configuration, and other non-standard conditions relating to mypy
+
+There is currently `a bug <https://github.com/python/mypy/issues/19548>`_
```

Comment:
Thanks, this is definitely a good idea to be explicit about exit codes, but let's agree on them first. I added a few observations in #19548, this test is inaccurate as written, and I doubt documenting *that* would be helpful...

Reply:
True. I'm Draft-ing this PR until we fix (or otherwise resolve):
* https://github.com/python/mypy/issues/15850
* #16764
* https://github.com/python/mypy/issues/19807
* https://github.com/python/mypy/issues/19808